contrib/sssd.spec.in

```diff
@@ -1032,7 +1030,7 @@ install -D -p -m 0644 contrib/sssd.sysusers %{buildroot}%{_sysusersdir}/sssd.con
 %config(noreplace) %{_sysconfdir}/krb5.conf.d/sssd_enable_passkey
 %endif
 
-%if 0%{?rhel}
+%if 0%{?fedora} >= 41 || 0%{?rhel}
 %pre common
 %if %{use_sysusers}
 %sysusers_create_compat contrib/sssd.sysusers
```

Comment:
This is not strictly related to the patch set, but `%sysusers_create_compat contrib/sssd.sysusers` does not work.

The current working directory at this point is the `rpmbuild` directory and both `BUILD` and `BUILDROOT` are already cleared at the the scriptlets are created. So I think the only solution is an additional `SOURCE` file as described in https://docs.fedoraproject.org/en-US/packaging-guidelines/UsersAndGroups/#_dynamic_allocation.

@pbrezina, this means basically to revert 736430aa0e .

bye,
Sumit

Reply:
Right, it didn't built, that's why I postponed this (listed in 'TODO').

Reply:
Hmm, I would bet that it did work for me. Anyway,source file is not needed and it would be the same problem anyway. We need to install the file to sysusersdir (which we do) and then use that one for %sysusers_create_compat.

Reply:
Since it is on the TODO list I'll close this thread as 'resolved'.